### DISCUSSION TITLE
Can't do a git pull because git wants me to sign in but signing in doesn't work

### DISCUSSION THREAD

**Discussion by randomuser11956** [community/community#62684] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Every time i want to do i git pull i'm greeted with this 
![image](https://github.com/community/community/assets/124638023/3d4a1e74-ce74-4815-a91c-6d042b2a23b0)
i tried making a token, using the token as password, using login with code, login in browser.... nothing works... i just can't do a git pull.

**Comment by priyankarpal** [comment]
sign in with your browser once more

**Reply by randomuser11956** [reply]
doesn't work, invalid username or password, authentication failed.

**Reply by priyankarpal** [reply]
yeah check your username & password

**Reply by randomuser11956** [reply]
they are good

**Comment by zMynxx** [comment]
Are you using GitHub Desktop application or just locally doing `git pull`? :open_mouth: 
If local, check your configuration, either manually view `~/.gitconfig` or use `git config -l`

**Reply by randomuser11956** [reply]
i just want to update an extension and when i go to the extension folder i do cmd, afterwards git pull and get that thing that i can't get rid of...
what do i modify when i use git config -1? i don't understand any of these i'm not a programmer :(

**Comment by randomuser11956** [comment]
i just did a git pull on a different program and it worked.. there's something wrong with this other program...
